### DISCUSSION TITLE
Question Regarding Licensing for Using the opcua-asyncio Library

### DISCUSSION THREAD

**Discussion by JaeHyeonSoon** [FreeOpcUa/opcua-asyncio#1767] [Q&A]
Hello,
If I write code using the opcua-asyncio library, should I follow the license of opcua-asyncio, or do I need to comply with the license of the OPC UA Foundation?

Thank you.

**Comment by AndreasHeine** [comment]
just -> https://github.com/FreeOpcUa/opcua-asyncio?tab=LGPL-3.0-1-ov-file#readme
